internal/keppel/ratelimit_driver.go

```diff
@@ -82,28 +84,28 @@ func RegisterRateLimitDriver(name string, factory func(AuthDriver, Configuration
 //implementation.
 type RateLimitEngine struct {
 	Driver RateLimitDriver
-	Store  throttled.GCRAStore
+	Store  *redis.Client
 }
 
 //RateLimitAllows checks whether the given action on the given account is allowed by
 //the account's rate limit.
-func (e RateLimitEngine) RateLimitAllows(account Account, action RateLimitedAction, amount uint64) (bool, throttled.RateLimitResult, error) {
+func (e RateLimitEngine) RateLimitAllows(account Account, action RateLimitedAction, amount uint64) (bool, *redis_rate.Result, error) {
 	rateQuota := e.Driver.GetRateLimit(account, action)
 	if rateQuota == nil {
 		//no rate limit for this account and action
-		return true, throttled.RateLimitResult{
-			Limit:      math.MaxInt64,
+		return true, &redis_rate.Result{
+			Limit:      redis_rate.Limit{Rate: math.MaxInt64},
```

Comment:
`Limit` does not make sense without an interval, so we should set it to be safe.
```suggestion
			Limit:      redis_rate.Limit{Rate: math.MaxInt64, Period: time.Second},
```